netlify-redirects

```diff
@@ -0,0 +1,5 @@
+# [Stage] Redirect default Netlify subdomain to primary domain
+https://mozilla-careers-stage.netlify.com/* https://careers.allizom.org/:splat 301!
```

Comment:
I thought this was handled automatically by Netlify when you added a custom domain. No?

Reply:
No it's suggested by them to add those lines when you add a domain. That's a redirect copy-paste from that suggestion box

Reply:
Cool. I must be thinking of the www redirect they offer. Thanks 👍